doc/Development_Documentation/20_Extending_Pimcore/13_Bundle_Developers_Guide/11_Adding_Object_Datatypes.md

```diff
@@ -29,8 +30,8 @@ is presented and an can be entered within Pimcore objects.
    It needs to extend `pimcore.object.tags.abstract`, be located in namespace `pimcore.object.tags` and named after the 
    `$fieldtype` property of the corresponding PHP class.
      
-   For examples have a look at the Pimcore core datatypes at  
-   [github](https://github.com/pimcore/pimcore/tree/10.x/bundles/AdminBundle/Resources/public/js/pimcore/object/tags)
+   For examples have a look at the Pimcore core datatypes at 
+   [github](https://github.com/pimcore/pimcore/tree/10.x/bundles/AdminBundle/public/js/pimcore/object/tags)
```

Comment:
The link broke due 2 facts: this https://github.com/pimcore/pimcore/pull/12356 change and the fact that `10.x` got renamed to `11.x`, so it would now redirecting to a 11.x branch where the Resources folder is removed.

In 10.5 branch, what about this change instead?
```suggestion
   [github](https://github.com/pimcore/pimcore/tree/10.5/bundles/AdminBundle/Resources/public/js/pimcore/object/tags)
```

Reply:
thanks - updated the link according to your solution